### ISSUE TITLE
メディカルAI専門コース＞1.4 実践編　CIFAR10＞accuracyの算出

### ISSUE THREAD

**Issue by naokitohda** [kikagaku/kikagaku-public-issues#33]
先日同様のissueを投稿しましたが、誤ってcloseしてしまったため、再度投稿致します。

accuracyの算出時、ValueErrorが発生します。最後のsoftmaxを省略しているため、predが0-1の範囲から外れることが原因と思われます。
accuracy算出用に別にsoftmaxを用意する方法を教えて頂き、一応正解することはできました。(バージョンはすべて最新のものを使用)

class Net(pl.LightningModule):

    def __init__(self):

        super().__init__()

　　#省略

        self.sm = nn.Softmax(dim=1)

    def forward(self, x):

       #省略

    def training_step(self, batch, batch_idx):

        x, t = batch

        y = self(x)

        loss = F.cross_entropy(y, t)

        pred = self.sm(y)

        self.log('train_loss', loss, on_step=False, on_epoch=True, prog_bar=True)

        self.log('train_acc', accuracy(pred, t), on_step=False, on_epoch=True, prog_bar=True)

        return loss

この点に関し、修正を頂けると幸いです。

**Comment by kikagaku** [comment]
ご報告いただき、ありがとうございます。この度はご迷惑をおかけしておりまして、失礼いたしました。

ご指摘頂いた問題含め、バージョン起因による問題は現在対応中です。対応完了次第ご案内いたしますので、ご不便をおかけしますが、お待ちいただけますと幸いです。

**Comment by kikagaku** [comment]
いつも学習していただきありがとうございます。
現在 Colaboratory にデフォルトで入っているパッケージバージョンに対応した資料にアップデートいたしました。
ご確認いただけますと幸いです。
